app/controllers/reservations_controller.rb

```diff
@@ -1,7 +1,26 @@
 class ReservationsController < ApplicationController
+  before_action :authenticate_user!, except: :create
+  before_action :set_reservation!, only: %i(destroy)
```

Comment:
I've commented below to remove this callback, with an explanation. But be consistent in your usage of the arrays syntaxes. Line above you use the standard literal hash `:create` and here, just a line below you use `%i(destroy)`. Both are good, just stick to one style.